dataprofiler/profilers/categorical_column_profile.py

```diff
@@ -81,6 +81,7 @@ def profile(self):
             profile["statistics"].update(
                 dict(categories=self.categories)
             )
```

Comment:
not your issue, but if you have to make another change in this pr, could you change this to match the format below:
`profile["statistics"]["categories"] =. self.categories`